.storybook/base/_helpers.scss

```diff
@@ -31,3 +31,10 @@
     flex-flow: row wrap;
   }
 }
+
+.storybook-root,
+.sb-story {
+  img {
+    width: 100%;
+  }
+}
```

Comment:
Can we move this into a different file to keep the base styles (copied from anatomy docs) separate?
Maybe in `.storybook/storybook.scss` which gets imported into global.